projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_line.g4

```diff
@@ -32,6 +32,13 @@ l_exec_timeout
    EXEC_TIMEOUT minutes = DEC seconds = DEC? NEWLINE
 ;
 
+l_length
```

Comment:
Broken, and must be resolved against #388

Reply:
fixed